collection/MCDM Productions; Strongholds and Followers.json

```diff
@@ -26752,7 +26752,7 @@
 							"entries": [
 								"Those who stand closest to the light cast the longest shadows."
 							],
-							"by": "Kālidāsa Dhar, Vizier to the Court of Virmastha",
+							"by": "Kalidasa Dhar, Vizier to the Court of Virmastha",
```

Comment:
The original was correct; please revert this

Reply:
Won't work with his Parser apparently. I'll let him know though